cartographer_ros/cartographer_ros/CMakeLists.txt

```diff
@@ -12,35 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-google_binary(cartographer_assets_writer
```

Comment:
We can't currently port assets_writer to ROS2 because it needs an implementation of rosbag to function.  I can leave this commented out if preferred.

I'll do that for now with a TODO comment.